tests/python/gaia-ui-tests/requirements.txt

```diff
@@ -1,13 +1,5 @@
 boto>=2.32.1
-marionette_driver==0.9
-marionette_client==0.16
-mozlog==2.11
-mozcrash==0.14
-moznetwork==0.26
-mozprofile==0.24
-mozrunner==6.7
-mozversion==1.2
-mozdevice==0.45
-moztest==0.7
+marionette_driver>=0.9
```

Comment:
Shouldn't we ping marionette_driver and marionette_client to those specific version numbers? An upgrade of those might cause incompatibility problems, otherwise?